bokeh/plotting_helpers.py

```diff
@@ -125,6 +125,7 @@ def _process_sequence_literals(glyphclass, kwargs, source):
         kwargs[var] = var
 
 def _make_glyph(glyphclass, kws, extra):
+        if extra is None: return None
```

Comment:
:) got it.

I think the code is ok. I was more concerned about the intention. I see that it's convenient and that's fine as _make_glyph is a private factory and is used in "conscious" way. I'd me negative about this if _make_glyph was more exposed.